src/routers/GetCornellMajorsRouter.ts

```diff
@@ -0,0 +1,31 @@
+import { Request } from 'express';
+import { SerializedCornellMajor } from '../common/types';
+import ApplicationRouter from '../utils/ApplicationRouter';
+import CornellMajor from '../entities/CornellMajor';
+import CornellMajorRepo from '../repos/CornellMajorRepo';
+
+class GetCornellMajorsRouter extends ApplicationRouter<
+  SerializedCornellMajor[]
+> {
+  constructor() {
+    super('GET');
+  }
+
+  getPath(): string {
+    return '/majors/';
+  }
+
+  async content(req: Request): Promise<SerializedCornellMajor[]> {
+    const callback = (
+      accum: SerializedCornellMajor[],
```

Comment:
instead of accum could thsi be like "cornellMajors" or at least "acc" cuz "accum" just doesn't seem appetizing oops

Reply:
+1 on cornellMajors

Reply:
Agreed but I should actually be using `map` instead of `reduce` with a callback anyway. Thanks for pointing that out